01-2024-schedule/week-1.md

```diff
@@ -0,0 +1,182 @@
+| BIP0032                                                            | [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)                                                                       |
+| Learn Bitcoin From The Command Line | [https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line](https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line)|
+| Bitstein - Setting up a bitcoin lightning network test environment | [https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a](https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a) |
+| Why Bitcoin Addreess is not like an account number | [https://sadeeqcode.medium.com/why-bitcoin-address-is-not-like-an-account-number-e5a5261a0605](https://sadeeqcode.medium.com/why-bitcoin-address-is-not-like-an-account-number-e5a5261a0605)|
+| Learn me a bitcoin | [https://learnmeabitcoin.com/](https://learnmeabitcoin.com/)|
+| Bitcoin Transaction tutorial | [https://github.com/chaincodelabs/bitcoin-tx-tutorial](https://github.com/chaincodelabs/bitcoin-tx-tutorial)|
+| BIP0039   | [https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)|
+| Wallet Recovery | [https://walletsrecovery.org/](https://walletsrecovery.org/)|
+| BIP0044   | [https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)|
+| BIP0049   | [https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki)|
+| BIP0084   | [https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki)|
+| BIP0086   | [https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki)|
+
+
+
+## Goals
+
+* Discuss and understand bitcoin philosophy
+* Become comfortable and proficient in setting up bitcoin developer environments
+* Be able to run and interface with Bitcoin Core and LND
+  * Understand signet, regtest and testnet.
+  * Connect Bitcoin Core and LND programmatically
+  * Interface with Bitcoin Core and LND using the CLI tools
+  * Programmatic control of bitcoind and LND
+  * Comfortable with running the full test suite(s) of Bitcoin Core
+* Build a toy bitcoin wallet using a library
+  * Add additional functionality of your choosing
+* Develop a robust understanding of bitcoin topics both in theory and in practice:
+  * Random number generation
+  * Keys and key material
+  * Derivation paths
+  * Address types
+  * The differences between keys, address and digital signatures
+* Gain experience with developing and using a Bitcoin wallet UI
+* Introduction to Technical Writing 
```

Comment:
I think Technical writing is amongst the general resources. I'm not sure it should be an objective.